### ISSUE TITLE
[Feature Request] Add a setting to disable downloading new Wine versions at start-up

### ISSUE THREAD

**Issue by hvcrios** [lutris/lutris#5162]
I'm not sure when this became default, but lutris auto-downloading new versions of wine-GE at start-up is not exactly ideal in my opinion.

I would like to request for there to be at least a setting to disable such behavior, even though it can be left as is as default.

I much prefer to download new wine versions when I deem necessary (basically if a game is working good with the version it's using, I see little reason to use another)

**Comment by danieljohnson2** [comment]
I've been working on this, with a lot of guidance of @strycore.

The master branch will still download new versions of wine, but it does it in the background while you game, instead of holding up the startup. That whole please-wait-starting-up dialog is gone now, for people using master.

I'll next work on settings to control what gets downloaded. @strycore has okayed a setting for Wine updates, but wants everyone *clearly warned* that if you are using some old crusty wine, we're not going to help you make that work.

Old wine = vinegar to us.

I'll keep this item open until the settings get merged.

**Comment by hvcrios** [comment]
Makes sense now that I think about it, I can see a bunch of people that don't quite understand how wine works complaining about a game not working, when all it wold take to make it work was using a newer wine build

Still I'd greatly appreciate that setting, so I thank you for working on it.

You could have wine updates enabled by default and then a pop-up show when the user disables it, warning them about old wine versions, that way no one could say they weren't warned.

But anyway, thx again.

**Comment by danieljohnson2** [comment]
Our plan is to delete your web browser once you flip that switch. It's the only way to be sure.

**Comment by hvcrios** [comment]
> Our plan is to delete your web browser once you flip that switch. It's the only way to be sure.

...🤔... eh, sounds reasonable to me!

**Comment by strycore** [comment]
> There have been cases when the solution was actually using an older wine build for me.

There is some severe misunderstanding on what we're doing. We're pushing Wine updates **because** we want to keep better track of regressions.
